backend/config/data_providers.yml

```diff
@@ -11,8 +11,6 @@ shared:
       subject: "Votre habilitation a été refusée"
     revoke: &shared_revoke_mailer
       subject: "Votre habilitation a été révoquée"
-    notify_submitted: &shared_notify_submitted_mailer
```

Comment:
you should remove this event from the doc also : at backend/app/views/enrollment_mailer/README.md

Reply:
I totally forgot to check the ReadMe ! Thank you